enterprise/server/remote_execution/executor/config/config.go

```diff
@@ -0,0 +1,83 @@
+package config
+
+import (
+	"flag"
+
+	"github.com/buildbuddy-io/buildbuddy/server/config"
+	"github.com/buildbuddy-io/buildbuddy/server/util/flagutil"
+)
+
+var (
```

Comment:
Is this an intermediate CL or why do we want these here instead of in the various libraries like we're doing with the app flags?